src/@apollo/client/core/ApolloClient__Core_Types.res

```diff
@@ -180,7 +180,7 @@ module MutationQueryReducersMap = {
     ~safeParse,
   ) =>
     Js.Dict.map(
-      (. mutationQueryReducer) => mutationQueryReducer->MutationQueryReducer.toJs(~safeParse),
+      (. mutationQueryReducer) => mutationQueryReducer->MutationQueryReducer.toJs(~safeParse, ...),
```

Comment:
I think this is the only place where new syntax is used that will force the minimum rescript version to 11? I'm not necessarily against it, but wonder if we should consider something like the below instead so we don't have to release a major version.
```
(. mutationQueryReducer) => (. json, options) => mutationQueryReducer->MutationQueryReducer.toJs(~safeParse, json, options),
```